Scan estimates in SLO info added (#4088)
SumoLogic/sumologic-documentation: docs/manage/partitions/flex/estimate-scan-data.md

Daily scan data will not be available if scheduled search runs only on selected days and not daily.
:::

## Scan estimates in SLO

When creating or modifying a log-based SLO, you can view the Daily scan and Yearly scan data information based on the SLO details by clicking the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" /> on top of the SLO right side panel.

Estimate the scan for different types of log-based SLOs:

- **Request-Based SLOs**. For request-based SLOs, data is scanned once per minute and stored in SLO storage (separate from logs storage). Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.

> - **Window-Based SLOs**: For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times from our logs storage.

Reviewer: ```suggestion
- **Window-Based SLOs**. For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times from our logs storage.
```

Daily scan data will not be available if scheduled search runs only on selected days and not daily.
:::

## Scan estimates in SLO

> When creating or modifying a log-based SLO, you can view the Daily scan and Yearly scan data information based on the SLO details by clicking the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" /> on top of the SLO right side panel.

Reviewer 2: ```suggestion
To view the Daily scan and Yearly scan data information based on your SLO details:

1. Go to the Monitoring and click the SLOs tab.
1. Hover over your SLO, click the three-dot kebab menu, and then click **Edit**. 
1. Click the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" /> in the upper right corner.
```

> Estimate the scan for different types of log-based SLOs:

Reviewer 2: ```suggestion
To estimate the scan for different types of log-based SLOs:
```

> - **Request-Based SLOs**. For request-based SLOs, data is scanned once per minute and stored in SLO storage (separate from logs storage). Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.

Reviewer 3: Can remove `(separate from logs storage)`

> - **Request-Based SLOs**. For request-based SLOs, data is scanned once per minute and stored in SLO storage (separate from logs storage). Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.

Reviewer 2: ```suggestion
- **Request-Based SLOs**. For request-based SLOs, data is scanned once per minute and stored in SLO storage. Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.
```

> Estimate the scan for different types of log-based SLOs:
> 
> - **Request-Based SLOs**. For request-based SLOs, data is scanned once per minute and stored in SLO storage. Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.
> 
> - **Window-Based SLOs**. For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times.

Reviewer 2: ```suggestion
- For **Request-based SLOs**, data is scanned once per minute and stored in SLO storage. Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.
- For **Window-based SLOs**, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times.
```

> - **Window-Based SLOs**. For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times from our logs storage.

Reviewer 2: ```suggestion
- **Window-Based SLOs**. For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times.
```

> - **Window-Based SLOs**. For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times from our logs storage.

Reviewer 3: Can remove `from our logs storage.` at the end.
